samples/msgext-ai-sentiment-analysis/nodejs/appManifest/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+   "manifestVersion": "1.16",
+    "version": "1.0.0",
+    "id": "${{TEAMS_APP_ID}}",
+    "packageName": "com.microsoft.teams.sentimentanalysis",
+    "developer": {
+      "name": "Microsoft",
+      "websiteUrl": "https://www.microsoft.com",
+      "privacyUrl": "https://www.microsoft.com/privacy",
+      "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+    },
+    "name": {
+      "short": "ME-ai-sentiment-analysis",
+      "full": "Sentiment analysis using message extension."
+    },
+    "description": {
+      "short": "Sentiment analysis Using Azure Open AI through messaging extension.",
+      "full": "This sample shows a feature where user can get sentiment analysis for posted messages in Teams chat using messaging extension and Azure Open AI."
+    },
+    "icons": {
+      "outline": "outline.png",
+      "color": "color.png"
+    },
+    "accentColor": "#60A18E",
+    "bots": [
+      {
+        "botId": "${{BOT_ID}}",
+        "scopes": [
+          "personal",
```

Comment:
Remove bot related code and other changes.

Reply:
Removed Bot Section and, when we create **ME** template toolkit sample by default it comes **bot id** placeholder will comes
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/109147706/7c0a1fab-b314-421c-9db3-92f67cb4d75c)